cmd/helm/downloader/manager.go

```diff
@@ -295,6 +311,22 @@ func (m *Manager) getRepoNames(deps []*chartutil.Dependency) (map[string]string,
 	// by Helm.
 	missing := []string{}
 	for _, dd := range deps {
+		// if dep chart is from local path, verify the path is valid
+		if strings.HasPrefix(dd.Repository, "file://") {
+			depPath, err := filepath.Abs(strings.TrimPrefix(dd.Repository, "file://"))
+			if err != nil {
+				return nil, err
+			}
+
+			if _, err = os.Stat(depPath); os.IsNotExist(err) {
+				return nil, fmt.Errorf("directory %s not found", depPath)
+			}
```

Comment:
Are there other errors we should check for here? Or a blanket `else if err != nil {return err }`? (same in `tarFromLocalDir`)